google-cloud-storage/samples/acceptance/storage_disable_bucket_lifecycle_management_test.rb

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LCC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "helper"
+require_relative "../storage_disable_bucket_lifecycle_management.rb"
+
+describe "disable_bucket_lifecycle_management" do
```

Comment:
Test can merged into bucket tests and don't need to be in a separate test file.

Reply:
Is this really better? @dazuma and I discussed this yesterday and our feeling was that since any grouping of tests is or would be somewhat arbitrary, a test-file-per-sample-file approach would be easiest.

Reply:
And as noted above, the test-file-per-sample-file organization is based on the [DLP samples and specs](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/df56ccb8258138e3d757505fec72523f5362ec28/dlp).

Reply:
The argument being that test-file-per-sample-file makes it easier to find the corresponding test for any given sample. (And Ruby's general convention is for there to be a one-to-one correspondence between test file and source file, for precisely that reason.)